src/tests/roc_pipeline/test_converter.cpp

```diff
@@ -46,8 +46,6 @@ TEST_GROUP(converter) {
         config.input_sample_rate = SampleRate;
         config.output_sample_rate = SampleRate;
 
-        config.internal_frame_size = MaxBufSize;
```

Comment:
Don't we need to compute config.internal_frame_size from MaxBufSize, as in other tests? I'd avoid relying on the default value here.

Reply:
We removed 'internal_frame_size' member from `ConverterConfig` and all other structures. Currently, we have only internal frame length, sample rate and channel mask. In line 49, I have already made an update for internal_frame_length.

Reply:
Ah, I see now.